libs/server/Metrics/Latency/GarnetLatencyMetricsSession.cs

```diff
@@ -80,7 +84,12 @@ public void ResetAll()
         public void Reset(LatencyMetricsType cmd)
         {
             int idx = (int)cmd;
-            metrics[idx].latency[PriorVersion].Reset();
+            disposeLock.WriteLock();
+            if(metrics != null)
```

Comment:
Minor issue with formatting.
```suggestion
            if (metrics != null)
```